Readme.md

```diff
@@ -700,6 +700,85 @@ test
 Environment variables can be configured for options of non-scalar types as well.
 In such case, the values of the environment variable will be split by `Path.PathSeparator` (`;` on Windows, `:` on Linux).
 
+### Suggest mode
+
+Suggest mode provides command-line autocompletion for Powershell and Bash. By default, it is disabled, but it can be enabled as follows:
+
+```csharp
+var app = new CliApplicationBuilder()
+    .AddCommandsFromThisAssembly()
+    .AllowSuggestMode(true) // allow suggest mode
+    .Build();
+```
+
+Once enabled, your shell must be configured to use suggest mode as follows:
+
+1. Add your application to the PATH
+2. Add a snippet to your shell's to instruct your shell in how to generate auto-completions. This can be done as follows.
+ 
+   ``` sh
+   > cmd [suggest] --install
```

Comment:
Can we drop `--install` from this and make it so `cmd [suggest]` (without other arguments) performs the install?